vital/types/bounding_box.h

```diff
@@ -215,6 +233,11 @@ bool operator== ( bounding_box<T> const& lhs, bounding_box<T> const& rhs )
 typedef bounding_box< int > bounding_box_i;
 typedef bounding_box< double > bounding_box_d;
 
+typedef std::shared_ptr< bounding_box_i > bounding_box_i_sptr;
+typedef std::shared_ptr< bounding_box_i const > bounding_box_i_cptr;
```

Comment:
(**API**) Can we please use `scptr` instead? Just `cptr` loses the `s` which indicates "shared pointer". (Also there is precedent for `scptr`; see `detected_object` and `image_container`.)